internal/controller/clusterdeployment_reconciler.go

```diff
@@ -162,15 +164,20 @@ func isOwnedByClusterInstance(ownerRefs []metav1.OwnerReference) bool {
 	return clusterInstanceOwner(ownerRefs) != ""
 }
 
-func (r *ClusterDeploymentReconciler) getClusterInstance(ctx context.Context, cd *hivev1.ClusterDeployment) (*v1alpha1.ClusterInstance, error) {
+func (r *ClusterDeploymentReconciler) getClusterInstance(
+	ctx context.Context,
+	cd *hivev1.ClusterDeployment,
+) (
+	*v1alpha1.ClusterInstance, error) {
```

Comment:
I really dislike this formatting. Especially how the parens don't match in the return value.
Maybe move the return value up a line?

Reply:
+1 looks like an unintentional line-break here. Fixed!